test/smoke/generated/purview-administration-rest/review/purview-administration-rest.api.md

```diff
@@ -561,6 +561,12 @@ interface ComplexReplacerConfigOutput {
     typeName?: string;
 }
 
+// @public (undocumented)
+function createClient(Endpoint: string, credentials: TokenCredential, options?: ClientOptions): PurviewMetadataPoliciesClient;
+
+// @public (undocumented)
+function createClient_2(endpoint: string, credentials: TokenCredential, options?: ClientOptions): PurviewAccountClient;
```

Comment:
This shouldn't be a top level export right? I'd expect it to only show up under the namespaces.

Reply:
I don't think these createClient functions are exported as toplevel, I only see 2 things exported as top level:

![image](https://user-images.githubusercontent.com/21109913/159593738-b20b8bf9-588c-464c-a82b-440869ff37ac.png)